backend/student_on_tour/views.py

```diff
@@ -78,11 +78,10 @@ def get(self, request, student_on_tour_id):
         """
         Get an individual StudentOnTour with given id
         """
-        stud_tour_instances = StudentOnTour.objects.filter(id=student_on_tour_id)
+        stud_tour_instance = StudentOnTour.objects.get(id=student_on_tour_id)
```

Comment:
`.get` will throw an error if it can't find an entry. Maybe use `.filter` and `.first`